ctags invalid option in Makefile

```
# ./scion.sh topology
Shutting down: Terminating this run of the SCION infrastructure
Shut down
Compiling...
ctags: invalid option -- 'L'
	Try `ctags --help' for a complete list of options.
Makefile:44: recipe for target 'tags' failed
make: *** [tags] Error 1
```

This was run in a fresh ubuntu 16.04 installation. I proceeded by commenting out `ctags` in the Makefile, which worked for now.

Continuous Integration might help to prevent such issues.

--- a/Makefile
+++ b/Makefile
@@ -1,14 +1,14 @@
-.PHONY: all clean goenv gogen vendor mocks bazel gazelle setcap tags
+.PHONY: all clean goenv gogen vendor mocks bazel gazelle setcap
 
 BRACCEPT = bin/braccept
 
 GAZELLE_MODE?=fix
 
-all: tags bazel
+all: bazel
 
 clean:
 	bazel clean
-	rm -f bin/* tags
+	rm -f bin/*
 	if [ -e go/vendor ]; then rm -r go/vendor; fi
 
 gogen:
@@ -39,6 +39,3 @@ gazelle:
 
 setcap:
 	tools/setcap cap_net_admin,cap_net_raw+ep $(BRACCEPT)
-
-tags:
-	which ctags >/dev/null 2>&1 || exit 0; git ls-files c | ctags -L -

